lib/cryptoexchange/exchanges/okex_swap/services/contract_stat.rb

```diff
@@ -0,0 +1,41 @@
+module Cryptoexchange::Exchanges
```

Comment:
this is a scenario where `open_interest` and `index` data  came from different url